pkg/util/converter/prom.go

```diff
@@ -662,7 +666,7 @@ func readMatrixOrVectorMulti(iter *jsonitere.Iterator, resultType string, opt Op
 				Type:   data.FrameTypeTimeSeriesMulti,
 				Custom: resultTypeToCustomMeta(resultType),
 			}
-			if opt.Dataplane && resultType == "vector" {
+			if opt.Dataplane && (resultType == "vector" || resultType == "scalar") {
```

Comment:
looking at this again, and i'm wondering if this is needed? do/should scalar results hit this function?

Reply:
It does not.